v-next/hardhat-toolbox-mocha-ethers/test/index.ts

```diff
@@ -11,7 +11,7 @@ describe("hardhat-toolbox-mocha-ethers", function () {
   describe("all the expected plugins are available", function () {
     useFixtureProject("toolbox");
 
-    it("should not throw because all the plugins should exist", async function () {
+    it.skip("should not throw because all the plugins should exist", async function () {
```

Comment:
```suggestion
    // TODO: This test should be brought back once an intermittent bug in EDR is resolved
    it.skip("should not throw because all the plugins should exist", async function () {
```